services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx

```diff
@@ -47,29 +54,161 @@ const dashboardViewWithReports = (
   </RouterWrappedComponent>
 );
 
-describe("Test Report Dashboard view (with reports, desktop view)", () => {
-  beforeEach(() => {
+describe("Test Report Dashboard view (Desktop)", () => {
+  beforeEach(async () => {
     mockedUseUser.mockReturnValue(mockStateUser);
     mockUseBreakpoint.mockReturnValue({
       isMobile: false,
     });
     mockMakeMediaQueryClasses.mockReturnValue("desktop");
-    mockedUseStore.mockReturnValue(mockReportStore);
   });
 
   afterEach(() => {
     jest.clearAllMocks();
   });
 
-  test("Check that WP Dashboard view renders", async () => {
-    await act(async () => {
-      await render(dashboardViewWithReports);
+  describe("Desktop view (basic)", () => {
+    test("Check that WP Dashboard view renders", () => {
+      mockedUseStore.mockReturnValue(mockReportStore);
+      render(dashboardViewWithReports);
+
+      expect(screen.getByText(wpVerbiage.intro.header)).toBeVisible();
+      expect(
+        screen.queryByText(wpVerbiage.body.table.caption)
+      ).toBeInTheDocument();
+      expect(screen.queryByText(wpVerbiage.body.empty)).not.toBeInTheDocument();
+      expect(screen.queryByText("Leave form")).not.toBeInTheDocument();
+    });
+  });
```

Comment:
does this need to be wrapped in "describe" if it's just a single test with no before or after logic?

Reply:
decided to consolidate this chunk, removed from described